Assets/BossRoom/Scripts/Game/Client/Character/ClientInputSender.cs

```diff
@@ -446,6 +447,11 @@ void Update()
                 RequestAction(ActionType.Emote4, SkillTriggerStyle.Keyboard);
             }
 
+            if (Input.GetKeyDown(KeyCode.Space))
```

Comment:
Is this documented in our popup tutorial?

Reply:
This can be done in the follow-up, visual enhancement PR.

Reply:
I don't see a PR, can you link the jira ticket or PR please? Just so this isn't lost.

Reply:
I've removed it as an action triggered through input. Instead it's contextual on the `targetId`. This enables this ability on mobile too. I've created a [task](https://jira.unity3d.com/browse/MTT-4304) to update the pickup action image with a proper one.